SharpPcap/TimestampType.cs

```diff
@@ -0,0 +1,17 @@
+﻿using System;
+namespace SharpPcap
+{
+    /// <summary>
+    /// Source of the timestamp
+    /// See https://github.com/the-tcpdump-group/libpcap/blob/master/pcap/pcap.h#L498 for details
+    /// </summary>
+    public enum TimestampType : int
+    {
+        Host,
+        HostLowPrecision,
```

Comment:
Assign the numbers
```suggestion
        HostLowPrecision = 1,
```

Reply:
maybe assign the first and let the compiler assign the rest? They are in order at the moment.

Reply:
better be explicit to prevent someone from adding one in the middle and shifting the value of the others.

Reply:
We'd be the ones adding it or reviewing it though. Let me make sure I link back to the source code where that array is defined. I guess I'm less concerned that we'll thing we can just reorder that list. The whole project is based upon a proper mapping of various fields to specific orders etc.